R/flatten.R

```diff
@@ -1,58 +1,53 @@
-#' Flatten part of a `dm` into a wide table
+#' Flatten a part of a `dm` into a wide table
 #'
-#' Gather all information of interest in one place in a wide table (on a database-[`dm`] a
-#' temporary table will be created).
-#' If referential integrity is given among the tables of the data model, the resulting
-#' table of this function will contain as many rows as the table `start` does (exceptions are
-#' `join = anti_join` (result is empty table with same columns as `start`) and `join = right_join`
-#' (number of rows equal to or larger than those of `start`)).
+#' Gather all information of interest in one place in a wide table - a temporary table that will be created on a database-[`dm`].
+#' If referential integrity is specified for the tables of the data model, the table that results from the flattening function
```

Comment:
I believe this is a remnant of a misconception we had earlier. Please replace by something like:

> Performs a disambiguation of column names and a cascade of joins.

The exact behavior is described below?

Reply:
```suggestion
#` Disambiguates column names and a cascade of joins.

Please see below for more details.
```